src/components/Modals/FiatRamps/fiatRampProviders/banxa.ts

```diff
@@ -58,7 +58,7 @@ export const createBanxaUrl = ({ assetId, address, action }: CreateUrlProps): st
    * since some Banxa assets could be on multiple chains and their default
    * chain won't be exactly the same as ours.
    */
-  params.set('blockchain', adapters.getBanxaBlockchainFromBanxaAssetTicker(asset))
+  params.set('blockchain', adapters.getBanxaBlockchainFromAssetId(assetId))
```

Comment:
Not clear the intent of this PR/what we're solving here, what do we mean by `our current mapping logic does not take into account the chain id when mapping to a Banxa chain id`?

The current `getBanxaBlockchainFromAssetId` method really only introspects `banxaTickerToAssetIdMap`  and `chainIdToBanxaBlockchainCodeMap` without any fancy logic, it's a simple mapping.
The fact we use a ticker as an input isn't a bug, it's a feature which acts as validation:
1. get the Banxa ticker from an `AssetId` using the `AssetIdToBanxaTickerMap` mapping
2. get a Banxa blockchain from said `AssetId`using the inverted `AssetIdToBanxaTickerMap`

I don't see how we don't `take into account the `ChainId` when mapping to a Banxa chain id` since an `AssetId` contains a `ChainId` part (see [CAIP-19 spec](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md))

Reply:
This does make sense after giving it a try at runtime now.
Would ideally prefer to keep the ticker logic, but indeed, this won't work for assets which exist as the same ticker across multiple networks.

Since this isn't relying on a ticker anymore, and the `AssetId` is already valid at this point, what about simplifying `getBanxaBlockchainFromAssetId` to `getBanxaBlockchainFromChainId`